### ISSUE TITLE
Last.fm support

### ISSUE THREAD

**Issue by theLastOfCats** [steve228uk/YouTube-Music#29]

**Comment by ivan-zakharchuk** [comment]
@steve228uk  Could you give me a hint about related code parts. I can try then to create a PR with this feature

**Comment by theLastOfCats** [comment]
@ivan-zakharchuk I knew how is made in GPM, but in Swift, u will need 3rd party lib for last.fm support, and also some kind of 'emitters' for correct scrobbling handling.

I think @steve228uk can guide you with emitters if there is implemented logic.

**Comment by walkerofmargins** [comment]
+1 on this - Would be awesome!

**Comment by treeshateorcs** [comment]
this is the only issue stopping me from using this program

**Comment by vitriolix** [comment]
another vote, i literally just found out about YT-Music because of googling for a Mac OS YTM client that supported scrobbling XD

**Comment by jtroutdev** [comment]
Add another +1 for me. The other app I was using supported it but not keyboard shortcuts... The desktop app does not support this application so external app monitoring seems to remain a non-option.

**Comment by DrKenReid** [comment]
Yep, I'd appreciate this feature, too. Spotify has a type of last fm login implemented for tracking scrobbling, this could too.
